lib/init/server.js

```diff
@@ -179,12 +179,16 @@ function emitConfigurationComplete() {
 }
 
 function initializeServers() {
```

Comment:
if you're not clustered, where are the various servers being started?

Reply:
I put cluster on this.cluster in the forkProcess function so this.cluster will always exist by this point. If you aren't using clustering, this.cluster.isMaster will always be true.

Reply:
K, a couple of things.

the assignment of properties on the class instance 'this' variable should follow the pattern of all instance property assignments.

Can you move any properties, assigned onto `this` to `lib/periodicClass.js` - this way they're always in the same place

Secondly, setting cluster.isMaster to true, if you're not using clustering is an anti-pattern. Can you check the settings property to see if a user is using clustering as an alternative?